docs/platforms/orange_pi3/orange_pi3.md

```diff
@@ -0,0 +1,85 @@
+# Edge Orchestration on Orange Pi 3
+
+[![Orange Pi 3](orange_pi3.jpg)](http://www.orangepi.org/Orange%20Pi%203/)
+
+## Preparing Orange Pi 3 board
+
+#### 1. Creating an image and writing Orangepi on a SD card
+1. Download Ubuntu image for Orange Pi 3 from (http://www.orangepi.org/downloadresources/).
+2. To install the Orangepi operating system follow the [instructions](http://www.orangepi.org/Docs/SDcardinstallation.html).
+
+> SD card must be at least 16 Gb
+
+#### 2. Start Orange Pi 3
+
+Insert the SD card into the Orange Pi 3 and turn on the power. Make configuration settings for the Orange the first time you turn it on.
+
+---
+
+## Quick start
+This section provides how to download and run pre-built Docker image without building the project.
+
+> TBD
```

Comment:
Better to indicate how initiate the starting point here to build up the HomeEdge on OrangePi. Any idea on this?